shared/src/authorization/authorizationClientService.test.ts

```diff
@@ -8,27 +8,14 @@ import {
   mockCaseServicesSupervisorUser,
   mockChambersUser,
   mockDocketClerkUser,
-  mockIrsPractitionerUser,
   mockJudgeUser,
   mockPetitionerUser,
   mockPetitionsClerkUser,
 } from '@shared/test/mockAuthUsers';
 
 describe('Authorization client service', () => {
   it('should return false when the user is undefined', () => {
-    expect(
-      isAuthorized(undefined, 'unknown action' as any, 'someUser'),
-    ).toBeFalsy();
-  });
-
-  it('should return true for any user whose userId matches the 3rd owner argument, in this case "someUser" === "someUser"', () => {
-    expect(
-      isAuthorized(
-        mockPetitionerUser,
-        'unknown action' as any,
-        mockPetitionerUser.userId,
-      ),
-    ).toBeTruthy();
```

Comment:
Before, `isAuthorized` allowed an optional `userId` to be passed (e.g., a user "isAuthorized" to see some case). However, this made authorization in general more complicated: we were doing similar checks in other function calls. Therefore, the `userId` parameter has been eliminated. See `CaseFactory.ts` for more on related refactoring.